src/main/converter/publication.ts

```diff
@@ -61,9 +63,29 @@ export class PublicationViewConverter {
             lcp: document.lcp,
             lcpRightsCopies: document.lcpRightsCopies,
 
-            // doc: r2Publication.Metadata,
+            duration: r2Publication.Metadata.Duration &&
+                this.convertSecondToDaysHoursMinutes(r2Publication.Metadata.Duration),
+            nbOfTracks: r2Publication.Metadata.AdditionalJSON?.tracks as number,
+
+            // doc: r2Publiction.Metadata,
 
             r2PublicationBase64,
         };
     }
+
+    private convertSecondToDaysHoursMinutes(seconds: number): ITimeDuration {
+        const days = Math.floor(seconds / (60 * 60 * 24));
+        seconds %= 60 * 60 * 24;
+        const hours = Math.floor(seconds / (60 * 60));
+        seconds %= 60 * 60;
+        const minutes = Math.floor(seconds / 60);
+        seconds %= 60;
```

Comment:
Just nit-picking about code style :)
Would you mind introducing variables? (to replace duplicated calculations and make the code more readable)
For example:

```
const secondsPerMinute = 60;
const minutesPerHours = 60;
const hoursPerDay = 24;
const secondsPerHour = minutesPerHours * secondsPerMinute;
const secondsPerDay = secondsPerHour * hoursPerDay;

const days = Math.floor(seconds / secondsPerDay);
seconds %= secondsPerDay;

const hours = Math.floor(seconds / secondsPerHour);
seconds %= secondsPerHour;

const minutes = Math.floor(seconds / secondsPerMinute);
seconds %= secondsPerMinute;
```